server/planning/utils.py

```diff
@@ -1,10 +1,81 @@
-from typing import Union
+from typing import Union, List, Dict, Any
 from bson.objectid import ObjectId
 from bson.errors import InvalidId
+from datetime import timedelta, datetime
+from flask_babel import format_time, format_datetime, lazy_gettext
+from eve.utils import str_to_date
+import arrow
 
 
 def try_cast_object_id(value: str) -> Union[ObjectId, str]:
     try:
         return ObjectId(value)
     except InvalidId:
         return value
+
+
+def get_formatted_contacts(event: Dict[str, Any]) -> List[Dict[str, Union[str, List[str]]]]:
+    contacts = event.get("event_contact_info", [])
+    formatted_contacts = []
+    for contact in contacts:
+        if contact.get("public", False):
+            formatted_contacts.append(
+                {
+                    "name": " ".join(
+                        [
+                            c
+                            for c in [
+                                contact.get("first_name"),
+                                contact.get("last_name"),
+                            ]
+                            if c
+                        ]
+                    ),
+                    "organisation": contact.get("organisation", ""),
+                    "email": ", ".join(contact.get("contact_email", [])),
```

Comment:
would keep that data as lists, it's easy to join it in the template, but might be harder to split it later